exonum-java-binding/core/src/main/java/com/exonum/binding/service/AbstractServiceModule.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.service;
+
+import com.google.inject.AbstractModule;
+
+/**
+ * A base class for {@link ServiceModule} implementations provided for convenience.
+ *
+ * <p>The implementation must be specified as an extension:
+ * <pre>
+ *   // fixme: do I really need to put explicit points if using abstract class?
+ *   &#64;Extension(points = ServiceModule.class)
```

Comment:
Let's put the code into ` <pre>{@code } </pre>` statement. It'll allow to use `@` instead of `&#64;`

Reply:
This post is saying it won't work :man_shrugging: (but I haven't launched the javadoc to check, must admit): https://reflectoring.io/howto-format-code-snippets-in-javadoc/

Reply:
🤕